### ISSUE TITLE
Better integration with ResourceTiming 

### ISSUE THREAD

**Issue by npm1** [w3c/navigation-timing#117]
In PerformanceTiming we use same-origin checks for security, whereas ResourceTiming uses TimingAllowOrigin checks. From the descriptions of PerformanceTiming, I'd expect all of the following to not require security checks in PerformanceNavigationTiming:

* domainLookupStart
* domainLookupEnd
* connectStart
* connectEnd
* secureConnectionStart
* requestStart
* responseStart

This seems to be fine per the processing model here https://w3c.github.io/navigation-timing/#processing-model. But if I click on the link to one of these, they take me to ResourceTiming, which does TAO checks for all of these (example: https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-domainlookupstart). This makes the spec reading quite confusing.

Especially confusing is secureConnectionStart because the processing model says "A user agent MUST also set the secureConnectionStart attribute as defined in the attribute's processing model in [RESOURCE-TIMING]." Does this mean that this attribute should require TAO checks even if PerformanceTiming exposed this attribute without same-origin checks?

**Comment by noamr** [comment]
I believe https://github.com/whatwg/html/pull/7105 fixes this. WDYT @yoavweiss & @npm1 ?

**Comment by yoavweiss** [comment]
I agree the confusion was solved by Fetch/HTML integration of both specs. (with the above PR and others).
@npm1 - thoughts?

**Comment by npm1** [comment]
I didn't look at it closely (and filed this two years ago), but I see the PR title is "Hide some navigation timing info when cross-origin redirects are present". Whereas this issue is about not requiring security checks for some of the attributes. I guess the answer is that we are actually wanting to be stricter in Nav Timing, in that we don't use TAO but instead only expose all of these when there are no cross-origin redirects in the navigation? Let me know if this understanding is correct or not!

**Comment by noamr** [comment]
> I didn't look at it closely (and filed this two years ago), but I see the PR title is "Hide some navigation timing info when cross-origin redirects are present". Whereas this issue is about not requiring security checks for some of the attributes. I guess the answer is that we are actually wanting to be stricter in Nav Timing, in that we don't use TAO but instead only expose all of these when there are no cross-origin redirects in the navigation? Let me know if this understanding is correct or not!

Yes exactly.

**Comment by npm1** [comment]
Alright I trust your call on this one! Feel free to close this issue when that PR lands :)

**Comment by noamr** [comment]
I double-checked this and it's now handled correctly.
